profiles/migrations/0036_set_fake_user_for_test_profiles.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-05-18 15:59
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def mark_all_fake_profiles(apps, schema_editor):
+    Profile = apps.get_model("profiles", "Profile")
+    Profile.objects.filter(user__username__startswith='fake').update(fake_user=True)
```

Comment:
maybe you should test for the string `fake.` (just to narrow even more the possibility of false positives.